server/src/test/java/io/crate/integrationtests/PgCatalogITest.java

```diff
@@ -173,8 +173,8 @@ public void test_pg_proc_select_variadic_and_non_variadic_functions() {
             "ORDER BY proname");
         assertThat(printedTable(response.rows()), is(
             "-395638146| array_difference| -1861355723| 1000.0| 0| false| false| true| 2277| [2277, 2277]| NULL| array_difference\n" +
-            "726540318| current_timestamp| -1861355723| 0.0| 0| false| false| false| 1184| [23]| NULL| current_timestamp\n" +
             "726540318| current_timestamp| -1861355723| 0.0| 0| false| false| false| 1184| []| NULL| current_timestamp\n" +
+            "726540318| current_timestamp| -1861355723| 0.0| 0| false| false| false| 1184| [23]| NULL| current_timestamp\n" +
```

Comment:
Why did this change?  Could it be that this is flaky because the `ORDER BY` is only on the name?

Reply:
CurrentTimestampFunction has now two signatures being registered, to cover the case `current_timestamp` as well as `current_timestamp(3)`. It is flaky in  the sense that if I change the order of signature registration, I have to change the test. I will make it resilient, thank you for spotting this.